### PR TITLE
add basic readme

### PR DESCRIPTION
**Pull request by BusyJay**
@queenypingcap @siddontang PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
any doc to install rust protoc and rust grpc?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# gRPC-rs
+
+[![Build Status](https://travis-ci.org/pingcap/grpc-rs.svg)](https://travis-ci.org/pingcap/grpc-rs)
+
+The Rust wrapper of [gRPC Core](https://github.com/grpc/grpc). [Grpc](http://www.grpc.io) is a high performance, open source, general RPC framework that puts mobile and HTTP/2 first.
+
+This project is still under developement, not all features are supported.
```
Comment (queenypingcap) [inline README.md]: Can we list which features we support and which features we don't?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- [x] interop test
+- [x] qps benchmark
+- [ ] custom metadata
+- [ ] health check
+- [ ] reflection
+- [ ] auth
+- [ ] load balance
+
+Prerequisites
+-------------
```
Comment (queenypingcap) [inline README.md]: Can we add the version information to these Prerequisites?

File: README.md (hunk 1 above)
Comment (BusyJay) [inline README.md]: Yes, as L12 ~ L22 shows.

Comment (queenypingcap) [inline README.md]: -> This project is still under development, not all features are supported. See [Status](#status) for more information.

File: README.md [hunk 3]
```diff
@@ -21,21 +21,53 @@ Status
 - [ ] auth
 - [ ] load balance
```
Comment (queenypingcap) [inline README.md]: Please spell out the abbreviations. For example, async, ssl, interop, auth... Please use uppercase for the first letter of each item.

File: README.md [hunk 4]
```diff
@@ -21,21 +21,53 @@ Status
...
+
+To generate the sources from proto files:
+
+1. Install protobuf compiler.
+
+```
+$ cargo install protobuf
+```
+
+2. Install grpc compiler.
```
Comment (queenypingcap) [inline README.md]: s/grpc/gRPC

**Pull request merged**